src/i18n/pl.json

```
@@ -0,0 +1,132 @@
+{
+  "Height": "Wysokość",
+  "Network": "Sieć",
+  "Main": "Główna",
+  "Development": "Deweloperska",
+  "Supply": "Zasoby",
+  "Market Cap": "Kapitalizacja rynkowa",
+
+  "All rights reserved": "Wszelkie prawa zastrzeżone",
+  "Version": "Wersja",
+  "Date": "Data",
+
+  "No Results": "Brak wyników",
+
+  "Transaction": "Transakcja",
+  "Transactions": "Transakcje",
+  "Confirmations": "Potwierdzeń",
+  "Reward": "Nagroda",
+  "Total Fee": "Łączna opłata",
+  "Total Forged": "Łącznie wytworzono",
+  "Total Amount": "Całkowita kwota",
+  "Timestamp": "Znacznik czasu",
+  "Generated by": "Wygenerowany przez",
+
+  "Block": "Blok",
+  "Block ID": "ID bloku",
+  "Previous block": "Poprzedni blok",
+  "Next block": "Następny blok",
+
+  "Price in": "Cena w",
+  "Day": "Dzień",
+  "Week": "Tydzień",
+  "Month": "Miesiąc",
+  "Quarter": "Kwartał",
+  "Year": "Rok",
+  "All time": "Wszystko",
+
+  "Menu": "Menu",
+  "Search": "Wyszukaj",
+  "Find a block, transaction, address or delegate": "Znajdź blok, transakcję, adres lub delegata",
+  "Nothing matched your search": "Brak wyników",
+  "Home": "Strona główna",
+  "Top Wallets": "Najbogatsze portfele",
+  "Delegate Monitor": "Monitor delegatów",
+
+  "Show more": "Pokaż więcej",
+
+  "Rank": "L.p.",
+  "Name": "Nazwa",
+  "Forged": "Wydobyto",
+  "Forged blocks": "Wydobytych bloków",
+  "Last Forged": "Ostatnie wydobycie",
+  "Status": "Status",
+  "Productivity": "Produktywność",
+  "Approval": "Poparcie",
+  "Delegates": "Delegaci",
+  "Delegate": "Delegat",
+  "Forged block recently": "Niedawno wydobyło blok",
+  "Missed block": "Przeoczyło blok",
+  "Not forging": "Nie wydobywa",
+  "Never forged": "Nigdy nie wydobywał",
+  "In queue for forging": "W kolejce do wydobycia",
+  "Last block": "Ostatni blok",
+  "from transactions": "z {count} transakcji",
+  "Total Forged (token)": "Łącznie wydobyto ({token})",
+  "Forging": "Wydobywa",
+  "Missing": "Przegapia",
+  "Missed block, Awaiting Slot": "Opuścił blok, Oczekuje na miejsce",
+  "Not Forging": "Nie wydobywa",
+  "Awaiting Slot": "Oczekuje na miejsce",
+  "Never": "Nigdy",
+
+  "Address": "Adres",
+  "Public Key": "Klucz publiczny",
+  "Balance (token)": "Saldo ({token})",
+  "Copied!": "Skopiowano!",
+  "Copy to Clipboard": "Skopiuj do schowka",
+  "QR Code": "Kod QR",
+  "Scan for Address": "Zeskanuj adres",
+  "Uptime": "Czas działania",
+  "Rank/Status": "Pozycja/Status",
+  "Blocks": "Bloki",
+  "missed": "przegapiono",
+  "Votes": "Głosy",
+  "Voters": "Wyborcy",
+  "Only voters with more than 0.1 Ark": "Tylko wyborcy posiadający więcej niż 0,1 Ark",
```

Necessary change to comply with #413 [@faustbrian]

```suggestion
  "Only voters with more than 0.1 token": "Tylko wyborcy posiadający więcej niż 0,1 {token}",
```